packages/ui/components/LabetText.tsx

```diff
@@ -0,0 +1,17 @@
+import { HStack, Text } from '@chakra-ui/react';
+
+import type { LabelProps } from './Label';
+import Label from './Label';
+
+type LabelTextProps = { content: string } & LabelProps;
+
+export default function LabelText({ labelTitle, content }: LabelTextProps) {
+  return (
+    <HStack spacing="0.5rem">
+      <Label labelTitle={labelTitle} />
+      <Text fontSize="sm" lineHeight="4" textTransform="uppercase">
```

Comment:
```suggestion
      <Text fontSize="sm" lineHeight={4} textTransform="uppercase">
```

마찬가지로 이런식으로 바꾸면 좋을 것 같아요!!